Intersect (Core)/Config/NpcOptions.cs

```diff
@@ -34,6 +34,48 @@ public class NpcOptions
         /// </summary>
         public bool ShowLevelByName = false;
 
+        /// <summary>
+        /// Configures whether or not the type of Npc is displayed adove to their name as a sprite tag.
+        /// [ Default is false ]
+        /// </summary>
+        public bool ShowNpcTags = false;
+
+        /// <summary>
+        /// NpcTags above the Npc's Name Labels, only works if ShowNpcTags = true
+        /// [ Default is true ]
+        /// </summary> 
+        public bool NpcTagsOnTop = true;
```

Comment:
since i decided to use switch instead, i did the following:
```
        /// <summary>
        /// Configures the position of the Npc Tags. Only works if ShowNpcTags = true.
        /// 0: NpcTags above the Npc's Name Label.
        /// 1: NpcTags under the Npc's Name Label.
        /// 2: NpcTags as prefix (left) of the Npc's Name Label.
        /// 3: NpcTags as suffix (right) of the Npc's Name Label.
        /// </summary>
        public int NpcTag_Position = 0;
```

will commit this draft soon with these changes and more